pipelines/tikv/migration/latest/pull_integration_test.groovy

```diff
@@ -89,7 +89,11 @@ pipeline {
                 axes {
                     axis {
                         name 'TEST_CASE'
-                        values 'autorandom availability', 'capture_session_done_during_task cdc_hang_on'
+                        values '"autorandom availability"', '"capture_session_done_during_task cdc_hang_on"', '"changefeed_auto_stop changefeed_error"',
+                            '"changefeed_fast_fail changefeed_finish"', '"changefeed_pause_resume changefeed_reconstruct"', '"cli cli_tls"', '"disk_full flow_control"',
+                            '"gc_safepoint http_api"', '"http_proxies kill_owner"', '"kv_client_stream_reconnect multi_capture"',
+                            '"processor_err_chan processor_panic"', '"processor_resolved_ts_fallback processor_stop_delay"', '"sink_hang sink_tls"',
+                            '"sorter stop_downstream"'
```

Comment:
May I ask, can we use `ls` to get all cases and assign them to `TEST_CASE`? @wuhuizuo @pingyu